configs/body/2d_kpt_sview_rgb_img/topdown_heatmap/coco/vipnas_coco.md

```diff
@@ -0,0 +1,39 @@
+<!-- [BACKBONE] -->
+
+<details>
+<summary align="right">ViPNAS (CVPR'2021)</summary>
+
+```bibtex
+@article{xu2021vipnas,
```

Comment:
For new papers, an item should be added here https://github.com/open-mmlab/mmpose/tree/master/docs/papers